### ISSUE TITLE
Multiple imports which include namespace can't be combined

### ISSUE THREAD

**Issue by antonovicha** [SonarSource/SonarTS#877]
**I want to report a bug.**

**SonarTS version:** 1.9 (build 3766)<br>
**Node.js version:** 8.11<br>
**TypeScript version:** 3.5.1<br>

**Rule key:** S3863

**Reproducer** 
```typescript
import * as express from "express";
import { Request, Response} from "express"; // False-positive here
```

**Expected behavior**

No error is reported. It's not possible to merge namespace and named imports. See similar issue in tslint: https://github.com/palantir/tslint/issues/3418

**Comment by quentin-jaquier-sonarsource** [comment]
Hello @antonovicha,
thanks for the feedback and the clear description.

In SonarTS 2.0, we are going to change this rule, and base it on [no-duplicate-imports](https://eslint.org/docs/rules/no-duplicate-imports), from ESLint.

The bad news is that this FP is also present in ESlint. Fixing it on our side does not make much sense, they could definitely benefits from this.
Therefore I advise you to report it to ESLint instead.

Best,
Quentin

**Comment by naingaungphyo** [comment]
> import * as express from "express";
> import { Request, Response} from "express"; // False-positive here
@antonovicha 
Why don't we access like **express.Request** , **express.Response** instead of importing 2nd line.
Is it something related with tree shaking or compiler(babel/webpack etc)/typescript version (before 2.7/after 2.7 etc) related case?
